akka-docs/src/main/paradox/typed/logging.md

```diff
@@ -435,10 +435,10 @@ All MDC properties as key-value entries can be included with `%mdc`:
 Akka is logging some events with markers. Some of these events also include structured MDC properties. 
 
 * The "SECURITY" marker is used for highlighting security related events or incidents.
-* Akka Actor is using the markers defined in @apidoc[akka.actor.ActorLogMarker].
-* Akka Cluster is using the markers defined in @apidoc[akka.cluster.ClusterLogMarker].
-* Akka Remoting is using the markers defined in @apidoc[akka.remote.RemoteLogMarker].
-* Akka Cluster Sharding is using the markers defined in @apidoc[akka.cluster.sharding.ShardingLogMarker].
+* Akka Actor is using the markers defined in @scala[@apidoc[akka.actor.ActorLogMarker$]]@java[@apidoc[akka.actor.ActorLogMarker]].
```

Comment:
I think it can even be:
```suggestion
* Akka Actor is using the markers defined in @apidoc[akka.actor.ActorLogMarker$].
```
or did that not work for you?